### DISCUSSION TITLE
Token usage tracking .

### DISCUSSION THREAD

**Discussion by AkashKulkarni4444** [microsoft/autogen#1615] [Q&A]
Is there a way to find out how many tokens have been used in a group chat ?
If there are multiple members in a chat then can we find out how many tokens have been use by each of them ?

**Comment by sonichi** [comment]
@qingyun-wu @kevin666aa @olgavrou if you know the answer could you please answer and consider adding to the documentation website?
cc @jackgerrits @olaoluwasalami
